microsoft-edge/devtools-guide-chromium/progressive-web-apps/index.md

```diff
@@ -36,7 +36,7 @@ See also [Overview of Progressive Web Apps (PWAs)](../../progressive-web-apps-ch
 
```

Comment:
typo: wither 
https://learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/widgets#widget-object

Reply:
Thanks for finding this, I'll create a follow-up PR.